ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java

```diff
@@ -157,6 +159,29 @@ void queryStorageUsageForHost(String clusterName, String hostname, ContainerStor
     }
   }
 
+  /**
+   * Delete container storage usage rows for given {@code clusterName}, {@code hostname}.
+   * @param clusterName The clusterName
+   * @param hostname The hostname
+   * @throws SQLException
+   */
+  void deleteStorageUsageForHost(String clusterName, String hostname) throws SQLException {
+    try (Connection connection = dataSource.getConnection()) {
+      try (PreparedStatement deleteStatement = connection.prepareStatement(deleteHostSql)) {
+        long startTimeMs = System.currentTimeMillis();
+        deleteStatement.setString(1, clusterName);
+        deleteStatement.setString(2, hostname);
+        deleteStatement.executeUpdate();
+        metrics.deleteTimeMs.update(System.currentTimeMillis() - startTimeMs);
+        metrics.deleteFailureCount.inc();
+      }
+    } catch (SQLException e) {
+      metrics.deleteFailureCount.inc();
+      logger.error("Failed to execute DELETE on {}, with parameter {}", ACCOUNT_REPORTS_TABLE, hostname, e);
```

Comment:
This log statement won't print the exception "e".

Maybe change this to 
```logger.error("Failed to execute DELETE on {}, with parameters {}, {} due to exception: {}", ACCOUNT_REPORTS_TABLE, hostname, clusterName, e);```

Reply:
nit: while we are at it, maybe change the `hostname` param to `hostName` (or `clusterName` to `clustername`).

Reply:
1. logger would check if the last argument is a throwable, if it's, logger would trim it out of the argument list and still print it.
2. sure, update clusterName to clustername.